pages/public_cloud/containers_orchestration/managed-rancher-service/getting-started/guide.en-gb.md

```diff
@@ -0,0 +1,276 @@
+| OVH API Endpoint        | Yes                                 | Select the OVHcloud subsidiary (EU, US, CA)                                                                                                                                  |
+
+  Minimum required access rights on the APIs routes (can be used to limit the access right when creating APIs keys on https://www.ovh.com/auth/api/createToken )
+
+  | Method              | API Route                                             |
+  |---------------------|-------------------------------------------------------|
+  | POST                | /cloud/project/{PROJECT-ID}/kube                      |
+  | POST/GET/PUT/DELETE | /cloud/project/{PROJECT-ID}/kube/*                    |
+  | GET                 | /cloud/project/{PROJECT-ID}/capabilities/kube/flavors |
+  | GET                 | /cloud/project/{PROJECT-ID}/capabilities/kube/regions |
+  | GET                 | /cloud/project/{PROJECT-ID}/network/private           |
+  | GET                 | /cloud/project                                        |
+
+4. Move to **Cluster Configuration**
+![MKS Driver Cluster Configuration](images/mksdriver_cluster_config.png)
+Provide the following parameters:
+  - the **Region** where your cluster will be deployed,
+  - the **Kubernetes Version**, only versions supported by Rancher current version are listed (you can refer to the [Official Support Matrix](https://www.suse.com/suse-rancher/support-matrix/all-supported-versions))
+  - the **Update Policy** you want to define for your managed cluster (for more information refer to the [Managed Kubernetes Update Policies](https://help.ovhcloud.com/csm/en-public-cloud-kubernetes-change-security-update?id=kb_article_view&sysparm_article=KB0049649) guide)
+
+
+5. Move to  **Network Configuration**
+![MKS Driver Cluster Configuration](images/mksdriver_network_configuration.png)
+
+  Provide the following parameters:
+  - the **Private Netword ID** for your MKS cluster. Use 'none' to create a cluster with nodes using Public IP or select an existing OVHcloud Public Cloud private network.
+  - the **Default vRack Gateway** (optional), leave it empty to use the default gateway of your Private Network.
+  - the **Private Network Routing As Default** state, activate this feature if you want to use an OVHcloud Managed Gateway or a custom Gateway as a single exit point for your MKS nodes.
+
+
+6. Move to **NodePools Configuration**
+![MKS Driver Cluster Configuration](images/mksdriver_nodepool_configuration.png)
+  Provide the following parameters:
+  - the **Name** of the NodePool
+  - the OVHcloud Instance **Flavor** used by this NodePool
+  - the **Autoscaling** state
+  - the **Size** of your NodePool (number of nodes that will be created)  
```

Comment:
if the autoscaling is enabled, it will display the min nodes and max nodes instead